synapse/module_api/errors.py

```diff
@@ -14,5 +14,9 @@
 
 """Exception types which are exposed as part of the stable module API"""
 
-from synapse.api.errors import RedirectException, SynapseError  # noqa: F401
+from synapse.api.errors import (  # noqa: F401
+    InvalidClientCredentialsError,
```

Comment:
Because `get_user_by_req` raises it if the token is invalid or doesn't belong to an existing user. Which makes me realise the docstring for the `get_user_by_req` method on the module API is lying, will fix.